modules/store/testing/spec/mock_store.spec.ts

```diff
@@ -541,81 +541,4 @@ describe('Cleans up after each test', () => {
       done();
     });
   });
-
-  it('should return the real value', (done: any) => {
-    TestBed.configureTestingModule({
-      imports: [
-        StoreModule.forRoot({} as any, {
-          initialState: {
-            value: 300,
-          },
-        }),
-      ],
-    });
-
-    const store = TestBed.inject(Store);
-    store.pipe(select(selectData)).subscribe((v) => {
-      expect(v).toBe(300);
-      done();
-    });
-  });
-});
```

Comment:
What's the reason that this test is removed?
I think it can be re-added and that it should pass.

Reply:
I think this is the reason why this hook has been introduced (more info : https://github.com/ngrx/platform/issues/2044 and https://github.com/ngrx/platform/pull/2115)
If selector are not reset, this test fail because selectData return 200, which is the value set in previous test ...

Reply:
Oh I didn't notice that, and you're right.
If that's the case, then I think we can completely remove the `Cleans up after each test` block.

Reply:
@timdeschryver I removed useless test.
I wonder if this PR does not introduce a breaking change. Some test that passed before will end up in error.

Reply:
Correct, this is indeed a breaking change.
But leaving it as is will also be a breaking change.
If I'm not mistaken, the suggested implementation has the same outcome removing this code.

That's why I'm suggesting to just remove it, which will already help projects and think of a good solution if someone bumps against this problem. I'm thinking that in the majority of the cases that this won't be a problem, I'm assuming that if you're mocking selectors in a test, that you're also mocking those selectors in the following test.

Reply:
Update: I verified that the same tests are failing with the suggested implementation:

```ts
      const TestBed = getTestBed() as any;
      const shouldResetSelectors = !TestBed.shouldTearDownTestingModule();
      if (shouldResetSelectors) {
        const mockStore: MockStore | undefined = TestBed.inject(MockStore);
        if (mockStore) {
          mockStore.resetSelectors();
        }
      }
```

Reply:
Yes, I had the the problem with the previous code, and removing the hook seems for me as well the best solution for now. 
Thanks for you help

Reply:
No, thank you! 🤗